osf/utils/migrations.py

```diff
@@ -161,6 +163,24 @@ def remove_schemas(*args):
     logger.info('Removed {} schemas from the database'.format(pre_count))
 
 
+class UpdateRegistrationSchemas(Operation):
+    """Custom migration operation to update registration schemas
+    """
+    reversible = True
+
+    def state_forwards(self, app_label, state):
+        pass
+
+    def database_forwards(self, app_label, schema_editor, from_state, to_state):
+        ensure_schemas(to_state.apps)
+
+    def database_backwards(self, app_label, schema_editor, from_state, to_state):
+        logger.info('Reversing UpdateRegistrationSchemas is a noop')
```

Comment:
Nitpicky: This should probably be a warning, e.g. using `warnings.warn`.

Not a blocker to merging, though.